tools/cli/help.txt

```diff
@@ -191,6 +191,7 @@ Options:
   --tailwind    Create a basic react-based app, with tailwind configured.
   --chakra-ui   Create a basic react-based app, with chakra-ui configured.
   --solid       Create a basic solid-based app.
+  --prototype   Create a prototype app.
```

Comment:
More explanation is needed here. This is too plain of a description.

Reply:
I've rewrote like this:
``Create a prototype app with the insecure & autopublish packages. Can be used along with other app commands`` what do you think?

Reply:
Maybe expand that they are for prototyping, but this is good as well.